pootle/apps/pootle_store/models.py

```diff
@@ -1273,6 +1274,50 @@ def live(self):
         """Filters non-obsolete stores."""
         return self.filter(obsolete=False)
 
+    def create_by_path(self, pootle_path, project=None,
+                       create_tp=True, create_directory=True):
+        from pootle_language.models import Language
+        from pootle_project.models import Project
+
+        lang, proj, dir, fn = split_pootle_path(pootle_path)
```

Comment:
Despite there are already other places where `dir` is used like this, IMHO it is better to use `dir_path` instead (or something similar) to avoid redefining `dir`.